src/App.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import './App.css';
+
+import users from './api/users';
+import todosFromApi from './api/todos';
+import { TodoList } from './components/TodoList';
+import { TodoForm } from './components/TodoForm/TodoForm';
+
+const preparedTodos = todosFromApi.map(todo => ({
+  ...todo,
+  user: users.find(user => (todo.userId === user.id)),
+}));
+
+class App extends React.Component {
+  state = {
+    todos: preparedTodos,
+  }
+
+  addTodo = (title, user) => {
+    this.setState((state) => {
+      const newTodo = {
+        userId: user.id,
+        id: state.todos.length + 1,
+        title,
+        completed: false,
+        user,
```

Comment:
No need to pass the whole 'user' object. You need only its name

Reply:
Small update:
Actually, we even don't need to pass the 'name' (only 'userId').
Each TODO in our list looks like this:
```
 {
    userId: 1,
    id: 1,
    title: 'delectus aut autem',
    completed: false,
  },
```
So we should pass 'userId' to our new TODO-object, as task description says:

> each TODO should have a title and userId selected from a list of given users

Also, you may pass the 'Name' of the user to the TodoList to make it looks prettier, but not to add it to the new 'todo' object and it's not essential for this task